nuxeo-core/nuxeo-core-mongodb/src/test/resources/OSGI-INF/mongodb-disable-renaming-listener-test.xml

```diff
@@ -0,0 +1,13 @@
+<?xml version="1.0"?>
+
+<component name="org.nuxeo.mongodb.duplicate.key.test.contrib" version="1.0">
+   <extension target="org.nuxeo.ecm.core.event.EventServiceComponent"
+    point="listener">
+    <listener name="duplicatedNameFixer"
```

Comment:
Why not just using `enabled=false` to disable the listener, as it's stated in the file name?

Worth changing the component name to reflect you are contributing to listeners?

Reply:
`enabled=false` -> 👍 

for the name generally i rename the component to avoid overriding any existing contribution

Reply:
AFAIK, you can also remove `class="org.nuxeo.ecm.core.model.DuplicatedNameFixer"`, the listener name is enough to be able to disable it.